ontology/spdx-ontology.owl.xml

```diff
@@ -354,7 +361,9 @@ Known issues:</rdfs:comment>
                 </owl:unionOf>
             </owl:Class>
         </rdfs:range>
-        <rdfs:comment xml:lang="en">The licensing information that was discovered directly within the package. There will be an instance of this property for each distinct value of alllicenseInfoInFile properties of all files contained in the package.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The licensing information that was discovered directly within the package. There will be an instance of this property for each distinct value of alllicenseInfoInFile properties of all files contained in the package.
+
+If the All Licenses Information from Files field is not present for a package and FilesAnalyzed property for that same pacakge is true or omitted, it implies an equivalent meaning to NOASSERTION.</rdfs:comment>
```

Comment:
Comments refer to the RDF terms, so use `licenseInfoFromFiles` instead of `All Licenses Information from Files` (`FilesAnalyzed` is correct)